firefox-ios/Client/Frontend/TrackingProtection/TrackigProtectionRedux/TrackingProtectionState.swift

```diff
@@ -94,120 +94,160 @@ struct TrackingProtectionState: StateType, Equatable, ScreenState {
 
         switch action.actionType {
         case TrackingProtectionMiddlewareActionType.clearCookies:
-            return TrackingProtectionState(
-                windowUUID: state.windowUUID,
-                trackingProtectionEnabled: !state.trackingProtectionEnabled,
-                connectionSecure: state.connectionSecure,
-                shouldClearCookies: true,
-                shouldUpdateBlockedTrackerStats: false,
-                shouldUpdateConnectionStatus: false,
-                navigateTo: .home,
-                displayView: nil
-            )
+            return clearCookiesState(from: state)
```

Comment:
I'd change all the methods name like in the other states to be: `handleClearCookiesAction` same for the others

Reply:
Thank you for pointing that out! I initially followed the pattern from the `defaultState` method when naming the others, but I see now that it doesn't align with the convention used in other states. My apologies for the oversight. I've updated the method names and rebased.

Reply:
No problem @ionixjunior that is not a big deal, it was just a nit comment